src/components/AppCard.tsx

```diff
@@ -12,12 +12,30 @@ export const AppCard = () => {
 		'specimen book. It has survived not only five centuries, but also the leap',
 		'into electronic typesetting, remaining essentially unchanged.',
 	];
-	return textArr.map((item, index) => (
-		<div className={styles.card} key={index}>
-			<div className={styles.editConteiner}>
-				<img src={editIcon} alt='edit' />
-			</div>
-			<h4 className={styles.text}>{item}</h4>
+	const [text, setText] = useState(textArr);
+
+	const reverseText = (index: number) => {
+		const newTextArr = [...text];
+		newTextArr[index] = newTextArr[index].split('').reverse().join('');
+		setText(newTextArr);
+	};
+
+	return (
+		<div>
+			{text.map((item, index) => (
```

Comment:
**issue:** Hm, I don't understand what is happening here - I assume that `AppCard` is a single Card component which should handle both sides (front, and back). Currently you're mapping over array of strings. So is it a list? Why? What are these values?